code/modules/projectiles/projectile/bullets/pistol.dm

```diff
@@ -412,9 +412,9 @@ Uranium, Contaminated
 
 /obj/item/projectile/bullet/mm14/uraniumtipped
 	name = "14mm uranium-tipped bullet"
-	damage = -5
+	damage = 24
 	armour_penetration = 0.1
-	irradiate = 300
+	irradiate = 30
```

Comment:
```suggestion
	irradiate = 80
```
Remember radiation armor still curbs these, you're giving up a nice amount of damage for 30÷(<100) radiation